libsrc/pce/clrscr.s

```diff
@@ -0,0 +1,32 @@
+
+        .include        "pce.inc"
+
+        .export _clrscr
+_clrscr:
+
+        st0     #VDC_MAWR
+        st1     #<$0000
+        st2     #>$0000
+
+        st0     #VDC_VWR
+        ldy     #$40
+rowloop:
+        ldx     #$80
+colloop:
+        lda     #' '
+        sta     a:VDC_DATA_LO
+        lda     #$02
+        sta     a:VDC_DATA_HI
+
+        dex
+        bne     colloop
+        dey
+        bne     rowloop
+
+        rts
```

Comment:
This function must put the cursor, also.  Therefore, that `rts` must be replaced with

~~~gas
; Go to the home position.

        stz     CURS_X
        stz     CURS_Y
        jmp     plot
~~~